include/erlang_ls.hrl

```diff
@@ -545,6 +545,21 @@
                                   , active_parameters => number()
                                   }.
 
+%%------------------------------------------------------------------------------
+%% Formatting
+%%------------------------------------------------------------------------------
+-type formatting_options() :: #{ tabSize           := integer()
```

Comment:
I'm trying this branch locally. Even if my client sends `2` as the `tabSize`, I get 4 spaces indentation...